pkg/operator/k8sutil/pvc.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package k8sutil for Kubernetes helpers.
+package k8sutil
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func CheckPVPresentInCluster(clientset kubernetes.Interface, drivers []string, clusterID string) (bool, error) {
```

Comment:
The k8sutil package is intended to not be specific to a storage provider. This method is really specific to checking for ceph-csi PVs, so i'd suggest just keeping it with the ceph controller. At least the `clusterID` property is specific to the ceph-csi storage class.